tests/integration/multimodal/yaml/embedding-multimodal-concatenate.py

```diff
@@ -2,21 +2,16 @@
 __license__ = "Apache-2.0"
 
 import numpy as np
-from jina.executors.decorators import batching, as_ndarray
 from jina.executors.encoders.multimodal import BaseMultiModalEncoder
 
 
 class ConcatenateMultiModalEncoder(BaseMultiModalEncoder):
 
-    batch_size = 10
```

Comment:
Keep it here as it will be used in the future

Reply:
ok